gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "10.2.6",
+  "version": "10.2.7",
```

Comment:
Does this count as a breaking change? @rpokorny

Reply:
> Does this count as a breaking change? @rpokorny

I wouldn't say so.  This combination was never intended to work, we're just being more explicit about it now since it was discovered that it caused _especially_ bad behavior

Reply:
> Ah thanks, that makes sense +1

I thought I left an additional comment on this but now I don't see it.  After looking through the PR diff I changed my mind.  Because we used to have an example of the combination that we are now prohibiting, effectively it was formerly supported.  So removing it would be a breaking change after all.

Reply:
> I thought I left an additional comment on this but now I don't see it. After looking through the PR diff I changed my mind. Because we used to have an example of the combination that we are now prohibiting, effectively it was formerly supported. So removing it would be a breaking change after all.

Bumped version: [5a68caf](https://github.com/sonatype/sonatype-react-shared-components/pull/729/commits/5a68cafba991e7ec342b701a762f9f14733198f9)

And per Ross's request, I updated this branch such that it merges to `FEATURE_11.0` once its approved.